src/components/App/TopNav.js

```diff
@@ -57,17 +58,20 @@ function CustomNavDropdown({ label, items }: ConfigMenuGroup) {
 
 const NAV_LINKS = [
   {
-    key: 'dependencies',
-    to: prefixUrl('/dependencies'),
-    text: 'Dependencies',
-  },
-  {
     key: 'search',
     to: prefixUrl('/search'),
     text: 'Search',
   },
 ];
 
+if (getConfig().depsMenuEnabled === true) {
```

Comment:
This can just be
```js
if (getConfig().depsMenuEnabled) {
```

Or is there a reason to fail if truthy but not strictly `true` ?

Reply:
no, updated